src/modules/dpos/validator/components/LatestVotes/index.js

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
+import React, { useMemo, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
-import Table from 'src/theme/table';
+import { QueryTable } from 'src/theme/QueryTable';
+import { useTransactions } from 'src/modules/transaction/hooks/queries';
+import { MODULE_COMMANDS_NAME_MAP } from 'src/modules/transaction/configuration/moduleCommand';
 import TransactionRow from '@transaction/components/TransactionRow';
 import header from './tableHeader';
+import { useDelegates } from '../../hooks/queries';
+import mergeUniquely from '../../utils/mergeUniquely';
 
-const LatestVotes = ({ votes, delegates }) => {
+const LatestVotes = ({ filters }) => {
   const { t } = useTranslation();
-  const canLoadMore = votes.meta ? votes.data.length < votes.meta.total : false;
+  const { data: delegates } = useDelegates();
+  const queryConfig = useRef({
+    config: {
+      params: {
+        ...filters,
+        moduleCommandID: MODULE_COMMANDS_NAME_MAP.voteDelegate,
+        sort: 'timestamp:desc',
+      },
+    },
+  });
 
-  const handleLoadMore = () => {
-    votes.loadData({ offset: votes.meta.count + votes.meta.offset });
-  };
+  const votedDelegates = useMemo(() => {
+    if (!delegates) return {};
+
+    const transformedResponse = mergeUniquely('username', delegates);
```

Comment:
is look like it 3th argument is missing do we need a merge here?
` mergeUniquely('username', oldData, newData);`